setup.py

```diff
@@ -75,14 +75,13 @@
         'requests>=2.0.0,<3',
         'PyJWT[crypto]>=1.0.0,<3',
 
-        'cryptography>=0.6,<4',
+        'cryptography>=0.6,<50',
```

Comment:
```suggestion
        'cryptography>=0.6',
```
The recommendation is to not set an upper bound:
https://github.com/pyca/cryptography/issues/5532#issuecomment-720252157